backend/src/handlers/alert.py

```diff
@@ -55,25 +55,25 @@ def get_alert(
         if alert is None:
             return JSONResponse(content={"message": "Alert not found"}, status_code=404)
 
-    filtered_alert = {}
+    filtered_alert: Dict[str, Union[str, int]] = {}
 
     if include is None or "text" in include:
         filtered_alert["text"] = alert.text
 
     if include is None or "start" in include:
-        filtered_alert["start"] = str(alert.start_datetime)
+        filtered_alert["start"] = int(alert.start_datetime.timestamp())
 
     if include is None or "end" in include:
-        filtered_alert["end"] = str(alert.end_datetime)
+        filtered_alert["end"] = int(alert.end_datetime.timestamp())
 
     return JSONResponse(content=filtered_alert)
 
 
 @router.post("/")
 def post_alert(req: Request, alert_model: AlertModel) -> JSONResponse:
     with req.app.state.db.session() as session:
-        dt_start_time = datetime.datetime.fromtimestamp(alert_model.start_time)
-        dt_end_time = datetime.datetime.fromtimestamp(alert_model.end_time)
+        dt_start_time = datetime.fromtimestamp(alert_model.start_time, timezone.utc)
+        dt_end_time = datetime.fromtimestamp(alert_model.end_time, timezone.utc)
```

Comment:
correct me if I'm wrong but it looks like we are assuming that time zone is always sent UTC, should we consider supporting passing a timezone in alert model? If so, I don't think it necessarily needs to be in this ticket but we should potentially make a new issue for it.

Reply:
It's a unix timestamp, so it should not really care about timezones. We're just interpreting it as UTC so postgres stores a timezone.